dynamic_values/network_acls.tf

```diff
@@ -10,7 +10,7 @@ locals {
         [
           # These rules cannot be added in a conditional operator due to inconsistant typing
           # This will add all cluster_rules if the acl object contains prepend_ibm_rules as true
-          for rule in local.cluster_rules :
+          for rule in local.rules :
```

Comment:
is that change necessary? where local.rules is defined?

Reply:
`cluster_rules` is the variable name for the acls with the static network_cidr and once the new acls are generated with the multiple cidrs the `cluster_rules` and the `app_rules `are concatenated into a single variable named `rules`